Gemfile

```diff
@@ -1,15 +1,4 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
-# runtime deps
-gem "gobject-introspection", "~> 3.0"
-
-# Add dependencies to develop your gem here.
-# Include everything needed to run rake, tests, features, etc.
-group :development do
-    gem "rspec", "~> 3.3"
-    gem "yard", "~> 0.8"
-    gem "redcarpet", "~> 3.3"
-    gem "github-markup", "~> 1.4"
-    gem "bundler", "~> 1.0"
-    gem "jeweler", "~> 2.0"
-end
+# all gems are handled by .gemspec (since this repo is for a gem)
+gemspec
```

Comment:
By deferring to `gemspec` it'll use the ones outlined here:
https://github.com/sandstrom/ruby-vips-1/blob/d740709ce58369af93e9d07f8d8eb912004f0567/ruby-vips.gemspec#L34-L40